config/default.yml

```diff
@@ -1433,6 +1433,11 @@ Lint/DisjunctiveAssignmentInConstructor:
   VersionAdded: '0.62'
   VersionChanged: '0.88'
 
+Lint/DuplicateBranch:
+  Description: Checks that there are no repeated bodies within `if/unless`, `case-when` and `rescue` constructs.
+  Enabled: pending
+  VersionAdded: '1.2'
```

Comment:
This should be 1.3.

Reply:
Ah, sorry, forgot to update this as well. So many new rubocop versions 😄

Reply:
We have been on 🔥 lately 😉